### ISSUE TITLE
Script validation misleading

### ISSUE THREAD

**Issue by ianleeder** [patrickdemooij9/SeoToolkit.Umbraco#163]
I created a script under Script Manager and forgot to enter a name.  The validation error highlights the script type, so I didn't notice my missing name for a few seconds.  
![image](https://user-images.githubusercontent.com/8241300/217687883-7ff8d9cd-8537-4bd7-be04-564c5196fb02.png)

**Comment by patrickdemooij9** [comment]
That is interesting. Will see what goes wrong there. Thanks for the report!

**Comment by patrickdemooij9** [comment]
I don't think this is an issue in the new backoffice anymore, hence I will close the issue
